styles/siteviews/layout.s2

```diff
@@ -52,36 +52,29 @@ function ItemRange::print(string{} opts) {
     if ($.all_subitems_displayed) { return; }
 
     var string anchor = $opts{"anchor"} ? "#$opts{"anchor"}" : "";
-    var string class = $opts{"class"} ? $opts{"class"} : "action-box";
+    var string class = $opts{"class"} ? $opts{"class"} : "";
 
-    """<div class="$class"><div>""";
+    """<div class="$class action-box"><div class="inner">""";
     print """<p style="font-weight: bolder; margin: 0 0 .5em 0;">""" + lang_page_of_pages($.current, $.total) + "</p>";
     var string url_prev = $this->url_of($.current - 1);
-    """<span class="page-prev">""";
-    if ($.current != 1) {
-        print " <b><a href='$url_prev$anchor'>$*comment_page_prev</a></b> ";
-    } else {
-        print " <b>$*comment_page_prev</b> ";
-    }
-    """</span>
-    <span style="text-align: center" class="page-links">""";
-    foreach var int i (1..$.total) {
-        if ($i == $.current) { "<b>[$i]</b> "; }
-        else {
-            var string url_of = $this->url_of($i);
-            "<a href='$url_of$anchor'><b>[$i]</b></a> ";
-        }
-        if ($i % 10 == 0){ """<br />"""; }
-        }
-    """</span>""";
-    var string url_next = $this->url_of($.current + 1);
-    """<span class="page-next">""";
-    if ($.current != $.total) {
-         print " <b><a href='$url_next$anchor'>$*comment_page_next</a></b> ";
-    } else {
-         print " <b>$*comment_page_next</b> ";
-    }
-    """</span>""";
+	"""<ul class="pagination">""";
+		    if ($.current != 1) {
```

Comment:
This is in the realm of nit, but since we don't have tidy for S2, I'm sorry :x

Can we fix the formatting here to be lined up with the code above/below, and only indent when we're going into a conditional/block of some kind?